quiche/src/stream.rs

```diff
@@ -1192,11 +1192,6 @@ impl SendBuf {
             self.fin_off = Some(max_off);
         }
 
-        // Don't queue data that was already fully acked.
-        if self.ack_off() >= max_off {
-            return Ok(data.len());
-        }
```

Comment:
@ghedo I think here would never happen, right? SendBuf.write only be called by application.